test/components/views/elements/AppTile-test.tsx

```diff
@@ -362,7 +362,7 @@ describe("AppTile", () => {
         it("should render", () => {
             const { asFragment } = renderResult;
 
-            expect(asFragment().querySelector(".mx_Spinner")).toBeFalsy(); // Wait until the loading spinner vanishes
+            expect(asFragment().querySelector(".mx_Spinner")).toBeFalsy(); // Assert that the spinner is gone
```

Comment:
```suggestion
            expect(container.querySelector(".mx_Spinner")).toBeFalsy(); // Assert that the spinner is gone
```

and add `container` to L363

Reply:
Thanks!